src/react_native/red_black_tree.cljs

```diff
@@ -0,0 +1,51 @@
+(ns react-native.red-black-tree
+  (:refer-clojure :exclude [remove update find])
+  (:require ["functional-red-black-tree" :as rb-tree]))
```

Comment:
In the spirit of our guidelines, perhaps we should use the alias as `red-black-tree`, there's no need to shorten the alias.